Ruby-tests/fibonacci_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'minitest/autorun'
+require './Fibonacci_numbers_par'
+require 'minitest/spec'
+
+describe FibonacciSums, 'Pars to limit fiibonacci' do
+  it 'is not a numbers' do
```

Comment:
'is not a numbers' -> esto significa que no es un numero, pero el test prueba que si es un numero